csv/ParameterDefs.csv

```diff
@@ -3598,3 +3598,7 @@ ZPLSC,ZPLSC,zplsc_values_channel_4,PD8079,array<quantity>,"[""channel_4_bins""]"
 OPTAA,,wavelength_a,PD8080,function,"[""-obs"",""wavelength""]",float32,,nm,-9999999,Optical Absorption Wavelength,6,,PFID16,"{""x"":""CC_awlngth""}",,,,The wavelength at which optical absorption measurements where taken in nm.,Optical Absorption Wavelength,Science Data,L2
 OPTAA,,wavelength_c,PD8081,function,"[""-obs"",""wavelength""]",float32,,nm,-9999999,Beam Attenuation Wavelength,6,,PFID16,"{""x"":""CC_cwlngth""}",,,,The wavelength at which beam attenuation measurements where taken in nm.,Beam Attenutation Wavelength,Science Data,L2
 OPTAA,,wavelength,PD8082,function,"[""-obs"",""wavelength""]",int32,,1,-9999999,Wavelength,6,,PFID215,"{""x"":""CC_cwlngth""}",,,,The wavelength dimension.,Wavelength,Science Data,L2
+ADCP_PD0,,non_zero_pressure,PD8083,function,,float32,,dbar,-9999999,Non-zero Pressure,3,,PFID216,"{""a1"":""PD710"", ""a2"":[""dpi_PRESWAT_L1"", ""None""], ""a1_scale_factor"":0.001}",,,,A non-zero pressure value.,,,L1
```

Comment:
I've never used a None option before. Pretty cool if it works. Do you have a test case for this?

Reply:
I have now added the test case in test/test_parameter_defs_csv.py.